static/flow/views/data-set-view.js

```diff
@@ -510,7 +509,7 @@ var DataSetView = function(options) {
         sequenceCount = 0;
 
         for (var i = 0; i < sequences.length; i++) {
-            if(sequences[i].name!="metadata"){
+            if (sequences[i].name != "metadata") {
```

Comment:
Unless you're expecting implicit conversions, seems like this should be `!==`.

Reply:
Agree - this seemed to pop up as a change from auto-formatting from VS code. A quick search indicates there are many more of these lurking in the codebase, alongside inconsistent indentation and more. I think it would be best to tackle this as a batch as part of next cleanup.

Reply:
Seconded - some of this could be part of a tech debt pass that is much-needed in Dataflow.